src/pages/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -0,0 +1,37 @@
+.about-mission-purpose{
+  font-weight: 400; 
+  text-align: left;
+
+  &__body{
+    margin-bottom: 75px;
+    font-size: 16px;
+    font-family: Poppins;
+    line-height: 24px;
+
+    h4 {
+      margin-top: 38px;
+      font-weight: bold;
+      font-style: normal;
+      font-size: 24px;
+      text-transform: uppercase;
+      }
+
+    li {
+      font-weight: normal;
+      font-style: normal;
+      font-size: 16px;
+      font-family: Poppins;
+      }
+
+    li + li{
+      padding-top: 12px;
+      } 
```

Comment:
Some of these arent needed - id prefer if we made a mixin for the h4 and import it into the h4 styling here.

Reply:
most of it is unnecessary. h4 in typography is set to the design system page in figma. Asked Andy whether he wants the design system styling or the one in phase 1. I will change to whatever he says.